db/migrate/20180424212252_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[5.1]
+  def change
+    create_table :users do |t|
+      t.string :first_name
```

Comment:
Consider adding not null constraints to some of these fields. Which ones do you want to always be present?